engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.io.File
+import java.net.URI
+
+class URIExt(uri: URI) {
+  def withFileSchemeDefault: URI = if (uri.isAbsolute) uri else new File(uri.getSchemeSpecificPart).toURI
+}
+
+object URIExt {
```

Comment:
You can just use `implicit class` instead: https://docs.scala-lang.org/overviews/core/implicit-classes.html

Reply:
Changed.